bucket/theia-ide.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.49.100",
+    "description": "A modern and open IDE for cloud and desktop. Theia platform based. Formerly “Theia Blueprint”.",
+    "homepage": "https://theia-ide.org/#theiaide",
+    "license": "EPL-2.0, GPL-2.0, MIT",
+    "notes": "Settings are stored in '%APPDATA%\\Theia IDE', and are not persisted by Scoop.",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.eclipse.org/downloads/download.php?mirror_id=1&file=/theia/ide/1.49.100/windows/TheiaIDESetup-1.49.100.exe#/dl.7z",
+            "hash": "sha512:a88a979fb1446cf463517377331ed67c78cdb9e2dec70e5d139b1b751883e2cecb20d45f060d9a1a8b30d7fb374d98ddd089e60c071cb5d8fd35819ba06a9156"
+        }
+    },
+    "extract_dir": "$PLUGINSDIR",
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\app-64.7z\" \"$dir\" -Removal"
+    },
+    "bin": "TheiaIDE.exe",
+    "shortcuts": [
+        [
+            "TheiaIDE.exe",
+            "TheiaIDE"
+        ]
+    ],
+    "checkver": {
+        "url": "https://download.eclipse.org/theia/ide/latest/windows/latest.yml",
+        "regex": "version: ([\\d\\.]+)"
```

```suggestion
        "regex": "version: ([\\d.]+)"
```

In regular expressions, a dot enclosed in brackets, '.', represents a literal dot and does not require a preceding '\\'.